src/main/resources/openapi.json

```diff
@@ -1,286 +1,935 @@
+                    "required": false,
+                    "schema": {
+                        "type": "integer"
+                    }
+                }
+            ]
+        },
+        "/consumers/{groupid}/instances/{name}/subscription": {
+            "post": {
+                "tags": [
+                    "Consumers"
+                ],
+                "description": "Subscribes a consumer to one or more topics.",
+                "operationId": "subscribe",
+                "requestBody": {
+                    "description": "List of topics to which the consumer will subscribe.",
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "$ref": "#/components/schemas/Topics"
+                            }
+                        }
+                    },
+                    "required": true
+                },
+                "responses": {
+                    "204": {
+                        "description": "No content."
+                    },
+                    "404": {
+                        "description": "The specified consumer instance was not found."
+                    },
+                    "409": {
+                        "description": "Subscription to topics, partitions, and pattern are mutually exclusive."
+                    }
+                }
+            },
+            "delete": {
+                "tags": [
+                    "Consumers"
+                ],
+                "description": "Unsubscribes a consumer from all topics.",
+                "operationId": "unsubscribe",
+                "responses": {
+                    "204": {
+                        "description": "No content."
+                    },
+                    "404": {
+                        "description": "The specified consumer instance was not found."
+                    }
+                }
+            },
+            "parameters": [
+                {
+                    "name": "groupid",
+                    "in": "path",
+                    "description": "ID of the consumer group to which the subscribed consumer belongs.",
+                    "required": true,
+                    "schema": {
+                        "type": "string"
+                    }
+                },
+                {
+                    "name": "name",
+                    "in": "path",
+                    "description": "Name of the consumer that you want to unsubscribe from topics.",
+                    "required": true,
+                    "schema": {
+                        "type": "string"
+                    }
+                }
+            ]
+        },
+        "/topics/{topicname}": {
+            "post": {
+                "tags": [
+                    "Topics"
+                ],
+                "description": "Sends one or more records to a given topic, optionally specifying a partition, key, or both.",
```

Comment:
"optionally specifying a partition" that's only available via a different endpoint AFAICS.